### PR TITLE
Add 'const' keyword

### PR DESCRIPTION
**Pull request by Peekmo**
Inexisting in haxe, I added the keyword `const` that is a shortcut for `public static inline var`.

Regards,
Peekmo

### CODE REVIEW

**Comment by deathbeam** [comment]
I also wanted to add this, but I was not really sure, because we are adding more reserwed keywords.

File: raxe/transpiler/CoreTranspiler.hx [hunk 1]
```diff
@@ -207,6 +207,10 @@ class CoreTranspiler implements Transpiler {
           handle.increment();
         }
       }
+      else if (handle.safeis("const")) {
+        handle.remove();
+        handle.insert("public static inline var");
```
Comment (deathbeam) [inline raxe/transpiler/CoreTranspiler.hx]: Add `handle.increment()` under this line please

**Comment by Peekmo** [comment]
```const``` keyword is meaningfull, and it helps code clarity no?

**Comment by deathbeam** [comment]
Yes, but in Haxe, you can also have inline static function, not only variable.

**Comment by Peekmo** [comment]
In fact, but "const" concept exists in many languages, and the translation in haxe could be "public static inline var", and it would be a nice shortcut. 

For inline static function, just keep things like that perhaps ? I don't really know if there's a name for this kind a functions..

**Comment by Peekmo** [comment]
As a side note, if you worry about reserved keywords, perhaps could we remove ```next``` ```do```, they do not add so much readability.

**Comment by deathbeam** [comment]
do is for anonymous functions, and next is because of more Ruby-like syntax.

**Comment by deathbeam** [comment]
Not going to add const keyword to not flood the namespace.

**Pull request closed without merging**